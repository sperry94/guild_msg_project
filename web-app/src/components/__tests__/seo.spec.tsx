import React from "react"
import { useStaticQuery } from "gatsby"
import renderer from "react-test-renderer"
import SEO, { SEOGraphqlQuery } from "../seo"

const mocked_useStaticQuery = useStaticQuery as jest.Mock<SEOGraphqlQuery>

describe("SEO", () => {
  beforeEach(() => {
    mocked_useStaticQuery.mockReturnValueOnce({
      site: {
        siteMetadata: {
          title: "My title",
          description: "My description",
          author: "Seth Perry",
        },
      },
    })
  })

  it("renders", () => {
    const tree = renderer.create(<SEO />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
