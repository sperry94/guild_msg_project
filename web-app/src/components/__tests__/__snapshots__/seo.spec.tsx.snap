// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO renders 1`] = `null`;
